features/scroll-behavior.yml

```
@@ -0,0 +1,9 @@
+name: scroll-behavior
+description: The `scroll-behavior` CSS property allows smooth scrolling for scroll actions not performed by the user, such as those triggered by navigation.
```

This definition works, but only if we were covering a specific declaration (`scroll-behavior: smooth`). Since this feature is a little more general, we ought to generalize to the full range of things it can do:

See also: https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#allows

```suggestion
description: The `scroll-behavior` CSS property controls whether scrolling is smooth or snaps, for scroll actions not performed by the user such as those triggered by navigation.
```